syntax.Rmd

```diff
@@ -230,6 +230,14 @@ list(
 
 Do not add extra spaces to places where space is not usually allowed.
 
+## Vertical space
+
+Use vertical whitespace sparingly, and primarily to separate your "thoughts" in your code, much like paragraph breaks in prose.
+
+* Avoid empty lines at the start or end of functions.
+* Only use a single empty line when needed to separate functions or pipes.
+* It often makes sense to put an empty line before a comment block, to help visually connect the explanation with the code that it applies to.
```

Comment:
My read is this implies "When a comment block is the first thing in a function body, a blank line is OK". Makes sense to me, but we may want to iterate the wording if that's not the intention.